### PR TITLE
Include transaction_hash For V2 Migration

### LINKED ISSUE #8340: Include transaction_hash in migration.sh
### Problem

We should use the migration script to copy `transaction_hash` from source to target

### Solution

Update script to include transaction_hash

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by jnels124**
**Description**:
* include `transaction_hash` in `migration.sh` 

**Related issue(s)**:

Fixes #8340

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by jnels124** [comment]
tables will process in this order: 
```
non_fee_transfer
 topic_message_lookup
 live_hash
 network_freeze
 transaction_child
 network_stake
 custom_fee_history
 reconciliation_job
 nft_allowance_history
 entity_bkup
 address_book_service_endpoint
 address_book_entry
 address_book
 crypto_allowance
 node_stake
 nft_allowance
 custom_fee
 prng
 token
 schedule
 token_allowance
 crypto_allowance_history
 entity_stake
 ethereum_transaction
 contract
 account_balance_file
 transaction_signature
 assessed_custom_fee
 token_account_history
 token_allowance_history
 token_history
 staking_reward_transfer
 contract_state
 sidecar_file
 entity_history
 nft
 nft_history
 token_account
 entity
 entity_stake_history
 file_data
 contract_transaction_hash
 token_transfer
 contract_state_change
 contract_transaction
 contract_log
 contract_action
 contract_result
 record_file
 topic_message
 token_balance
 transaction_hash
 account_balance
```

**Review by steven-sheehy — APPROVED** [review]

**Review by xin-hedera — APPROVED** [review]
LGTM

File: docs/database/README.md [hunk 1]
```diff
@@ -266,20 +266,26 @@ since open access until April 2023.
...
-Following are the prerequisites and steps for migrating V1 data to V2. As of mirror node v0.103.0, the migration script is expected to migrate full mainnet data in 7 days.
+Following are the prerequisites and steps for migrating V1 data to V2. As of mirror node v0.103.0, the migration script is expected to migrate full mainnet data in 10 days.
 
-1. Create a Citus cluster with enough resources (Disk, CPU and memory). For GKE, an e2-custom-6-32768 can be used.
-2. Ensure the source and target schemas are compatible by deploying the same version to both. Be sure to leave importer disabled for the target deployment.
-3. Get the correct version of [flyway](https://flywaydb.org/documentation/usage/commandline/) based on your OS and
+1. Make sure `bc` and `csvkit` are installed on the machine where the migration script will be run. For Ubuntu, you can install them using the following commands:
+   ```shell
+   sudo apt-get install bc
+   sudo apt-get install csvkit
```
Comment (steven-sheehy) [inline docs/database/README.md:274]: nit
```suggestion
   sudo apt-get install -y bc csvkit
```

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by xin-hedera — DISMISSED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**